src/Maker/MakeEntity.php

```diff
@@ -203,11 +203,8 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             $generator->writeChanges();
         }
 
-        if (
-            !$this->doesEntityUseAnnotationMapping($entityClassDetails->getFullName())
-            && !$this->doesEntityUseAttributeMapping($entityClassDetails->getFullName())
-        ) {
-            throw new RuntimeCommandException(sprintf('Only annotation or attribute mapping is supported by make:entity, but the <info>%s</info> class uses a different format. If you would like this command to generate the properties & getter/setter methods, add your mapping configuration, and then re-run this command with the <info>--regenerate</info> flag.', $entityClassDetails->getFullName()));
+        if (!$this->doesEntityUseAttributeMapping($entityClassDetails->getFullName())) {
+            throw new RuntimeCommandException(sprintf('Only attribute mapping is supported by make:entity, but the <info>%s</info> class uses a different format. If you would like this command to generate the properties & getter/setter methods, add your mapping configuration, and then re-run this command with the <info>--regenerate</info> flag.', $entityClassDetails->getFullName()));
```

Comment:
This is the one that we need to keep and determine if 
1) AttributeMapping is _not_ used
2) If somehow - AttributeMapping was being used but the class had `@ORM\Entity..` type annotations.
Then we would throw this exception..